src/main.cpp

```diff
@@ -562,8 +562,8 @@ int alphabeta(Position &pos,
         if (!in_check && alpha == beta - 1) {
             // Reverse futility pruning
             if (depth < 5) {
-                const int margins[] = {0, 50, 100, 200, 300};
-                if (static_eval - margins[depth] >= beta) {
+                const int margins[] = {50, 50, 100, 200, 300};
```

Comment:
I forgot to update this when rebasing locally and it may have as much or more Elo.